backend/common/src/main/java/ai/verta/modeldb/common/CommonConstants.java

```diff
@@ -17,6 +17,12 @@ public interface CommonConstants {
   String LIQUIBASE_MIGRATION = "LIQUIBASE_MIGRATION";
   String RUN_LIQUIBASE_SEPARATE = "RUN_LIQUIBASE_SEPARATE";
   String BACKEND_PID = "verta-backend.pid";
+  String FILENAME = "FileName";
```

Comment:
this looks like it's being used for a very specific purpose, i.e. a query parameter. Recommend suffixing it with something that identifies what it's being used for (like "FILENAME_QUERY_PARAMETER"). Also, is this "common" enough to be in this pool of constants that all of our java services will be accessing?

Reply:
`String FILENAME = "FileName";` is used for multiple purpose like you said as query parameter also used in artifact store endpoint path url. So i have make it just FILENAME instead of duplicating with two like
```
String FILENAME_QUERY_PARAMETER = "FileName"
String FILENAME_ARTIFACT_FILE_PATH_PREFIX = "FileName"
```

Reply:
> is this "common" enough to be in this pool of constants that all of our java services will be accessing?
Yes it is used across multiple backends.

Reply:
I would prefer to have two different constants if they are being used for two different purposes. Otherwise, it's too easy to break one when you change for the other.

Reply:
Fixed